src/gmail.js

```diff
@@ -2390,5 +2440,19 @@ var Gmail = function(localJQuery) {
     return false;
   }
 
+  /**
+   * Track the change of emails per page
+   * 
+   */
+  api.observe.on(
```

Comment:
I think it would be best to not activate any observer methods unless the user starts it. Hijacking http calls should be necessary on when its needed by the user. You should clarify that this needs to run along with the get emails endpoint.